packages/s3-credentials-endpoint/index.js

```diff
@@ -226,9 +226,10 @@ distributionApp.use((err, req, res, _next) => {
   return res.boom.badImplementation('Something broke!');
 });
 
+const server = awsServerlessExpress.createServer(distributionApp);
```

Comment:
```suggestion
const server = awsServerlessExpress.createServer(distributionApp);

```
nit - feel free to disregard